app/components/resource/ResourcesTableItem.js

```diff
@@ -33,17 +33,19 @@ class ResourcesTableItem extends Component {
       nextReservation,
       resource,
     } = this.props;
+    const { closes, opens } = getOpeningHours(resource);
+    const now = moment();
+
+    if ((now < moment(opens)) || (moment(closes) < now)) {
```

Comment:
- [ ] If resources are closed for the whole day their openinghours are `{ closes: null, opens: null }`, for example the [3d printer](http://localhost:3000/resources/auungfvj7qba/reservation).

So you'll have to add a check for that as well. Something like:
```javascript
if (!closes || !opens) {
  return <td className="resource-table-row availability reserved">Suljettu</td>;
}
```